### ISSUE TITLE
[PTR Raid report - Mount Hyjal]: Solaris May 18th

### ISSUE THREAD

**Issue by IskrenPlashkov** [chromiecraft/PTR-review#11]
### Mechanics comments on trash mobs

Giant Infernal - He can be enslaved, however when dismissed he is immediately gone. To make things worse, this does not count towards progress of the wave - aka you can no longer kill the trash wave in time and have to wait for the time to expire for next trash to spawn. **BUG**

Every other trash mob is fine.

### Tuning comments on trash mobs

Everything is OK

### Mechanics comments on Rage Winterchill

Everything is OK

### Tuning comments on Rage Winterchill

His HP might be a bit much, especially for more casual groups.

Most importantly the Icebolt dmg is more managable now. He is still more likely than not to kill the target, however healers at least have a chance to react. It acts as a soft enrage of sorts. Personally I am ok which such hard mechanic however I can easily see how many others would be frustrated, and cant say I blame them. Perhaps another small nerf would be nice here, especially since this can overlap with D&D or nova dmg. One last thing, currently this spell **can crit.** Imho this is a no-no. This makes it unavoidable one-shot and such mechanics are horrible for raid experience.

### Mechanics comments on Anetheron

Carrion swarm still sends projectile animation one way but dmg in a completely different direction - **BUG**
Sleep seems to be affecting everyone in 55 yard radius from the initial target. This leads to massive disruption of the raid that can not be prevented. You simply can not spread the raid far enough to only have few ppl eat it. It can also hit the main tank which is incorrect - **BUG**

The sleep is really breaking the fight right now. It was not like this in the first iteration of the raid. I find conflicting info on how this mechanic is supposed to be. 1 source says it affects everyone in small range of the primary target, other says it just hits 3 random players (highest threat excluded). Whichever one you choose to implement is fine and will make the fight actually doable, which at this point is nowhere near.

### Tuning comments on Anetheron

Infernals simply have too much hp. Ranged can not kill them in time before the next 1 spawns and eventually raid gets overwhelmed. Boss hp is also too much so its not like we can kite them all and nuke boss. The immense added disruption from sleep makes neither of these possible.

### Mechanics comments on Kaz'rogal

Everything is OK

We fucked around with resetting him and found out that his mark of kaz'rogal counter was not resetting after you kill him once. This made him cast the ability with the smallest possible cd - 10 seconds from the very start, making the fight completely impossible. Now this is likely because we used GM commands to kill and respawn him, but please make sure this does not occur from regular wipes as well. Because if so, he will be completely impossible to kill if you wipe on him once.

### Tuning comments on Kaz'rogal

Boss hp is a bit too much. We had a very well geared group today and while it appeared possible to kill him normally, it was by the skin of our teeth. Any mistake or simply less geared group would not make it.

### Mechanics comments on Azgalor

All mechanics work on paper, however something must be done with timers overlapping.  Will elaborate below as this has a lot to do with tuning as well.

### Tuning comments on Azgalor

Boss hp a bit too much. Raid will consistently lose people to doom so small nerf to his hp would be nice for normal raid groups.

His AA now hits way too much. We tried tanking with bis gear pala, war and bear, all of which had full hots and more often than not they would die within the 5 second silence (pala would pop ardent but you get the idea). While you could in theory stack all possible buffs on the tank - all hots from multiple druids, priest shield, shaman physical resistance etc to guarantee the tank surviving, it is unrealistic to expect everything to be up the whole time for 9-10 minute fight.

Rain of fire dmg is now almost manageable and healers have time to rect. The big problem however is that it very often overlaps with silence. This means unless the target has enough hp to survive the whole 5 second dot its dead. 

With current tuning silence is the big problem of the fight. As it is unavoidable mechanic simply reducing its cd will not work - it will just delay a big number of ppl dying (likely including the main tank). I strongly suggest nerfing his AA and rain of fire dmg a bit.

Last but not least doomguards have too much hp. Same issue as Anether infernals really, ranged simply dont have enough time to kill the add down before next one spawns, however here it is much worse as there is the constant threat of death from RoF.

### Mechanics comments on Archimonde

Doomfire now spawns directly on its primary target - **BUG** 
Given the fact that its dmg is now double what it should be this is guaranteed 1 shot. The fire should spawn 10- yards away from boss and chase towards its target. If the target somehow survives (atm this is only possible with adding extra hp via gm command) it still moves way too quickly so its animation needs to be slowed down,

### Tuning comments on Archimonde

Boss hp is way too much. 19 mil (I know its effectively 90% of that but you get the idea) is not doable at all given how easy it is to die. 1 death is almost guaranteed wipe even for the most hardcore group. It is very hard to imagine every1 surviving long enough to do 17,2 m dmg with this tuning.

His AA is now bit too strong. It was strong before, but now it is hard to imagine the tank being kept alive for the insanely long fight.

By far the worst offender here is the doomfire. I dont know what you guys did but now it does over double its regular dmg. 5k+ on initial dmg followed by 5k+ first tick, which happens instantly btw. IF you somehow survive this shortly after you get hit for 5k more on the 2nd tick. 

From what I have seen the initial dmg should be 2400, with every tick afterwards being reduced by 150. There should be 3 seconds between each tick.

### Overall comments on raid mechanics

Sadly I think the raid is in a more buggy state then before. Anetheron started relatively well scripted but now the fear makes it a fiesta. Carrion swarm is also bugged but at least that one can be healed through.

The other massive issue is Archimond doomfire spawning directly ontop of its target. Even the the tuning is removed this put immense pressure on healers, who will already be struggling to keep the main tank up.

Last but not least I have to mention Azgalor fear again. It is not bugged and the whole fight in theory works, however with RoF overlapping so frequently with fear it becomes unavoidable death combo.

### Give a rating out of 10 for the state of the mechanics in the raid

5

### Overall comments on tuning

Unfortunately the raid is not ready at all and I think last tuning changes were a step in the wrong direction. Fact of the matter is our hp has not changed much (if at all) compared to TBC values so having spells buffed so much makes most of them unavoidable 1 shots. The small nerfs to icebolt and RoF were good but not worth the price of much much stronger boss AA. Azgalor is the shining example of this when now most tanks wont survive the first silence.

Basically right now only 1st boss is doable, and only for very strong pumper groups who can push past the soft enrage of ppl dying to icebolts. Kaz'rogal in theory too but you cant get past Anetheron.

### Give a rating out of 10 for the state of the tuning in the raid

2

**Comment by elthehablo** [comment]
Thanks for the report. Will make some changes in tuning based on this
